src/types/verification.ts

```diff
@@ -26,28 +26,32 @@ export interface IVerification {
   formId: IFormSchema['_id']
   expireAt: Date
   fields: IVerificationFieldSchema[]
+  paymentField: IVerificationFieldSchema
 }
 
 export type UpdateFieldData = {
+  isPayment: boolean
```

Comment:
Per my other comment, `isPayment` is actually a "property" of `fieldId`, so the structure feels a little redundant... We can think about this more and change that later though (it's all internal after all), so I reckon we should merge first and have the task done for now 🤔

Reply:
Yea it does feel a bit weird, but then if we just use the `fieldId`, we'll have a lot more if blocks. Alternatively, I think we can take out the boolean flag as another argument for the function (`updateHashForField`) - do you think that will make more sense?

Reply:
Okay I'm just gonna leave this to a separate PR if we wanna change it hahaha